manifest.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
```

Comment:
I prefer to have a folder with all the needed yamls. I think the organization is better and the readability as well.

Reply:
For example, we can have a k8s folder and inside we can have a deployment forlder, service and so on. And inside of each one the yamls. app_deployment.yaml, postgresql_deployment.yaml...

Reply:
It's fine for me, both settings are OK

Reply:
Done, it was isolated by functional groups (not by k8s kinds)